File: Eclipse-Workspace/PLT_Exercises/src/Power.java
import java.util.*;

public class Power {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter The First Number (Base:X) : ");
		int X = sc.nextInt();
		System.out.print("Enter The Second Number (Exponent:n) : ");
		int n = sc.nextInt();
		int power = 1;
		for (int i = 1; i <= n; i++) {
			power = power * X;
		}
		System.out.println(X + "^" + n + " : " + power);
	}

}

File: Training-Workspace/Web Development/Day_Four_Tasks/power.js
const prompt = require("prompt-sync")();

let result = 1;

let base = prompt("Enter The Base: ");
let exponent = prompt("Enter The Exponent: ");

for (let index = 0; index < exponent; index++) {
  result *= base;
}

console.log(`${base}^${exponent} = ${result}`);
